src/components/Utils/order/processOpenOrderError.js

```diff
@@ -11,7 +11,7 @@ const ERROR_CODES = {
   accessProviderNotFound: 'accessProviderNotFound',
 };
 
-const showUpdateOrderError = async (response, callout) => {
+const processOpenOrderError = async (response, callout, openModal) => {
```

Comment:
why do you think that `process` is better than `show` ? basically show tells me that it show an error, but process is kind of confusing, since there is no any processing really, just showing modals or notifications. It could be OK if you were naming it initially as `process*`, but I don't see any strong points to rename in this case. Do you?

Reply:
returned to the previous name, but my point was we provide errors 2 different ways and call callbacks according to the error. Modal window  isn't showed by the method